design/work_item_children.go

```diff
@@ -0,0 +1,23 @@
+package design
+
+import (
+	d "github.com/goadesign/goa/design"
+	a "github.com/goadesign/goa/design/apidsl"
+)
+
+var _ = a.Resource("work-item-children", func() {
```

Comment:
@aslakknutsen I've put the [`list-source-link-types` action](https://github.com/almighty/almighty-core/blob/478f50127c81bea88da6523f203108ba2a042d50/design/resources.go#L58) in the `workitemtype` resource. It returns a list of WITs. The repository functions that are called by the WIT controller on the other hand come from the work item link repositories.

That's why I would create a `list-children` action in the work item resource.